vmm/src/config.rs

```diff
@@ -407,11 +407,26 @@ impl Default for RngConfig {
 pub struct FsConfig {
     pub tag: String,
     pub sock: PathBuf,
+    #[serde(default = "default_fsconfig_num_queues")]
```

Comment:
This is horrible!

Reply:
Do you want me to change the name? Or are you complaining about serde itself :)

Reply:
We could also implement the `Default` trait for `FsConfig`, and initialize a mutable one from the `parse` function.
Both `PathBuf` and `String` implement the `Default` trait already, so it would only be about the remaining fields.

Reply:
We don't really want the default from `tag` and `sock`, and from a `serde` perspective, if the field `num_queues` is not provided for instance, it will still rely on the `serde` macros to get the right default value. It won't be able to get only one default value out of the `Default` implementation: https://serde.rs/attr-default.html